proposals/0295-codable-synthesis-for-enums-with-associated-values.md

```diff
@@ -290,7 +290,7 @@ would encode to:
 ```json
 {
   "lade": {
-    "schluessel": "MyKey"
+    "key": "MyKey"
```

Comment:
Thanks for spotting this. Using `"schluessel"` here was intended, but the `enum StoreCodingKeys` above should instead be `LoadCodingKeys` and `_1` removed.

Reply:
@puretears ping